lib/check-dependencies.js

```diff
@@ -202,7 +202,36 @@ const checkDependenciesHelper = (syncOrAsync, config, callback) => {
             return;
         }
 
-        const depVersion = require(depJson).version;
+        const depJson = require(depJsonPath);
+
+        // Support NPM package aliases
```

Comment:
This is not only supported by npm, originally it was a Yarn geature. So we shouldn't restrict that to `npm`.

I know Yarn isn't tested right now but I'd like to change it so let's not lock in to internal npm stuff.

Reply:
I'd put Yarn firmly in "not supported" category. But sure, added support for yarn aliases

Reply:
Why not? It's a popular package manager and the alias feature is based on Yarn design.